src/core/state/rainbowChains/index.ts

```diff
@@ -20,7 +22,7 @@ export interface RainbowChain {
 
 export interface RainbowChainsState {
```

Comment:
We should be able to remove migrations here since `handleDiffAndMergeRainbowChains` is handling the diffing and we actually haven't had a breaking change (only appended networks since beginning of history) to me knowledge.